omero/developers/Python.rst

```diff
@@ -375,6 +431,28 @@ Read-Annotate groups, this will include other users' data - see
     image = conn.getObject("Image", imageId)
     print "Image: ", image,
 
+- **To save an Object's owner as another user (Admins only)**
```

Comment:
"To save an Object's owner as another user (Admins only)"  - not sure what this says. Does it want to change the owner of a given object ? Then maybe reformulate